src/styles/blocks/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+
+  border: 1px solid $color-border;
+
+  padding-top: 32px;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  &__image {
+    padding-left: 19px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__info {
+    margin-left: 16px;
+  }
+
+  &__title {
+    margin: 40px 0 4px;
+    color: $color-text-black;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    margin: 0;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $color-text-gray;
+  }
+
+  &__group1 {
+    margin-top: 16px;
+
+    @include centred-flexbox;
+  }
+
+  &__group1-reviews {
+    margin: 0 16px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $color-text-black;
+  }
+
+  &__price {
+    margin: 24px 16px 15px 0;
+
+    line-height: 18px;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: baseline;
+  }
+
+  &__price-text {
+    margin: 0;
+    font-size: 12px;
+    font-weight: 400;
+    color: $color-text-gray;
+  }
+
+  &__price-value {
+    margin: 0;
+    font-size: 16px;
+    font-weight: 700;
+    color: $color-text-black;
+  }
+
+  &:hover {
```

Comment:
This hover effect should be smooth in both cases, when mouse enter and when it leaves